dev-java/opentest4j/opentest4j-1.3.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source test"
+MAVEN_ID="org.opentest4j:opentest4j:1.3.0"
+JAVA_TESTING_FRAMEWORKS="junit-4"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Open Test Alliance for the JVM"
+HOMEPAGE="https://github.com/ota4j-team/opentest4j"
+SRC_URI="https://github.com/ota4j-team/opentest4j/archive/r${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-r${PV}"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+
+DEPEND=">=virtual/jdk-11:*"
+RDEPEND=">=virtual/jre-1.8:*"
+
+JAVA_SRC_DIR=( src/{main,module}/java )
+
+JAVA_TEST_GENTOO_CLASSPATH="junit-4"
+JAVA_TEST_SRC_DIR=( src/test/java )
+JAVA_TEST_RESOURCE_DIRS=( src/test/resources )
+
+src_test() {
+	JAVAC_ARGS="--source 11 --target 11" java-pkg-simple_src_test
```

Comment:
i guess `--target 11` makes it not runnable on `jre-1.8` as specified in `RDEPEND`. also it is not obvious here why it needs `jdk-11` for compilation, so i can't tell from this whether version 11 is also needed for runtime or runtime is fine with version 1.8.

Reply:
> i guess `--target 11` makes it not runnable on `jre-1.8` as specified in `RDEPEND`. also it is not obvious here why it needs `jdk-11` for compilation, so i can't tell from this whether version 11 is also needed for runtime or runtime is fine with version 1.8.

we have `jdk-11` for compilation for `module-info.java` and runtime seems fine with `1.8`.
similar to `gson-2.11.0-r2` with `--source 17 --target 17`.

Reply:
well, according to documentation `--target 11` (or `--target 17`) generates bytecode of that java version, and such a bytecode cannot be run on a lower versions of java, for obvious reasons.